tests/tests/test_mtls.py

```diff
@@ -123,32 +122,17 @@ def hsm_setup(self, pin, ssl_engine_id, device):
 
     def hsm_get_key_uri(self, pin, ssl_engine_id, device):
         pt11tool_output = device.run(
-            "export SOFTHSM2_CONF=/softhsm/softhsm2.conf; p11tool --login --provider=/usr/lib/softhsm/libsofthsm2.so --set-pin="
+            "p11tool --login --provider=/usr/lib/softhsm/libsofthsm2.so --set-pin="
             + pin
             + " --list-all-privkeys"
```

Comment:
Once we are here, what about modernizing the formatting?
```suggestion
            f"p11tool --login --provider=/usr/lib/softhsm/libsofthsm2.so --set-pin={pin} --list-all-privkeys"
```